CONTRIBUTING.md

```diff
@@ -2,78 +2,97 @@
 
 Hi! We're really excited that you are interested in contributing to ACPaaS UI. Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
 
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Issue Reporting Guidelines](#issue-reporting-guidelines)
-- [Pull Request Guidelines](#pull-request-guidelines)
-- [Development Setup](#development-setup)
-- [Project Structure](#project-structure)
+- [ACPaaS UI Contributing Guide](#acpaas-ui-contributing-guide)
+  - [Issue Reporting Guidelines](#issue-reporting-guidelines)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Development Setup](#development-setup)
+  - [Project Structure](#project-structure)
+  - [Releases](#releases)
 
 ## Issue Reporting Guidelines
 
-- Always use [GitHub Issues][github-issues] to create new issues.
+- Use the [#acpaas-ui channel](https://dgpls.slack.com/messages/C4M60PQJF) on slack to ask questions about using ACPaaS UI.
+- Use the [#acpaas-ui-dev channel](https://dgpls.slack.com/messages/C4S2D7KTK) on slack to discuss changes to the ACPaaS UI code itself.
+- Use [GitHub Issues][github-issues] to report bugs, request features, ask policy questions or propose policy changes.
+
+  > Note: policy questions are about the way ACPaaS UI is developed and released, or about its architecture. We use github issues to have a public archive of these discussions.
+  >
+  > Maintainers: check the [issue handling guidelines](./guidelines/ISSUES.md) for how to handle reported issues.
 
 ## Pull Request Guidelines
 
-- The `master` branch is basically just a snapshot of the latest stable release. All development should be done in dedicated branches. **Do not submit PRs against the `master` branch.**
+- The `master` branch is where we ship releases from. Always send PR's to `master`.
 
-- Checkout a topic branch from the relevant branch, e.g. `dev`, and merge back against that branch.
+- Commit your changes to a topic branch with a name matching these examples:
+  - `feature/more-cowbell`
+  - `fix/broken-cowbell`
+  - `docs/document-cowbell`
+  - For a list of suggested keywords, check the [commit message convention](./guidelines/COMMITS.md).
 
-- Work in the `src` folder and **DO NOT** check in `dist` in the commits.
+- Work in the `packages` folder and **DO NOT** check in `dist` or `examples` in the commits.
```

Comment:
It should be a mandatory step to check in examples. The example folders are used on: https://digipolisantwerp.github.io/acpaas-ui_angular/modules/avatar

Every change or new feature should be added as an example

Reply:
I'll improve the wording. I meant the top-level examples folder which is a build artifact from the styleguide app. Maybe that folder should be placed below dist or renamed to dist-examples.